src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -93,15 +94,20 @@
     private String parentCommit;
     private Collection<Path> paths = new HashSet<>();
     private boolean authorOrCommitter;
+    private boolean retainFullCommitSummary;
 
+    public GitChangeSet(List<String> lines, boolean authorOrCommitter) {
+        this(lines, authorOrCommitter, isTruncateTitle());
+    }
     /**
      * Create Git change set using information in given lines
      *
      * @param lines change set lines read to construct change set
      * @param authorOrCommitter if true, use author information (name, time), otherwise use committer information
      */
```

Comment:
New argument needs a javadoc comment.  Apparently I don't have a CI process that is checking the javadoc for warnings, since this should have generated a javadoc warning.

```suggestion
     * @param retainFullCommitSummary if true, do not truncate commit summary in the 'Changes' page\
     */
```